### ISSUE TITLE
limit on entity size?

### ISSUE THREAD

**Issue by janeuhd** [COATnor/COAT_portal_issues#45]
@frafra what is the limit to file size for a single file in the test instance? I try to test uploading a raster (TIF) but get errors related to file size too large. I use 'realistic' file sizes (e.g. last one I tested was 10 MB). If I know the limit I can crop a test data set that meets these for now.

**Comment by frafra** [comment]
There is always a limit. For the test instance the maximum size has not been changed, so it meant that the default is 1 MB. I brought it to 100 MB. That change should be enough to allow you using bigger files.

Details: this setting is not specific to CKAN nor the extensions we developed, but it is related to the web server that is put on front of all the web services running on the same server; as I did a new deployment on a non-NINA machine, I just left the default settings for the web server.

Technical details: for the NGINX web server, the option to set is `client_max_body_size` https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size
